ci-operator/jobs/infra-periodics.yaml

```diff
@@ -60,7 +60,7 @@ periodics:
           cpu: 500m
     serviceAccountName: upgrader
 - agent: kubernetes
-  cluster: app.ci
+  cluster: build10
```

Comment:
Were we advised to switch to `build10`?

Reply:
there was a job `ci/prow/ordered-prow-config` failure that recommended to switch to build10. This PR will require DPTP team approval, let me ask them to take a look first

Reply:
@nmars got in approval from Bear, please ltm when you get the chance